website/src/components/Messages/MessageTableEntry.tsx

```diff
@@ -34,9 +47,17 @@ interface MessageTableEntryProps {
   message: Message;
   enabled?: boolean;
   highlight?: boolean;
+  avartarPosition?: "middle" | "top";
+  avartarProps?: AvatarProps;
```

Comment:
These props are just to make the new UI compatible with the current UI. Will be cleanup in next PR